packages/backend/src/open_payments/payment/outgoing/lifecycle.ts

```diff
@@ -81,6 +81,14 @@ export async function handleSending(
     finalReceiveAmount: maxReceiveAmount
   })
 
+  deps.telemetry
+    ?.getOrCreate('transactions_total', {
+      description: 'Count of funded transactions'
+    })
+    .add(1, {
+      source: deps.telemetry.getServiceName()
```

Comment:
The setting of `source` (and probably calling of the `add`) should be encapsulated in the telemetryService, the services that use this as a dependency shouldn't need to worry about setting `source: deps.telemetry.getServiceName()` every time

Reply:
On another look at this, the source is not something all metrics send.
The object that gets sent as the second parameter when calling `add` represents the additional properties that will be attached to the metric.
I think this should be controlled externally, as it is now, by the place that creates the metric through the `getOrCreate`method.

It is just the case on this particular metric that it sends the 'source' which happens to be a property of the telemetry service itself through DI, but the additional properties object could hold anything else.

Reply:
>  the source is not something all metrics send.

In which cases would it be better not setting `source`? (If `source` would always just be telemetry service, maybe we can even leave it out?)

Reply:
the source is not always telemetryService. the serviceName in telemetry Service is the config.instanceName of rafiki.
This is so that we have the counts of transactions with distinction between instances in the network in the final dashboards.
I dropped the source attribute in the transaction_amount metric for privacy concerns, but i'd rather keep it here, on the count metric